streampark-flink/streampark-flink-submit/streampark-flink-submit-core/src/test/scala/org/apache/streampark/flink/submit/test/ParameterTestCase.scala

```diff
@@ -39,7 +43,31 @@ class ParameterTestCase {
     Assertions.assertEquals(param.get("flink.home"), "hdfs://nameservice1/streampark/flink/flink-1.11.1")
     Assertions.assertEquals(param.get("app.name"), "testApp123")
     Assertions.assertEquals(param.get("flink.deployment.option.parallelism"), "5")
-
   }
 
+  @Test def testExtractProgramArgs(): Unit = {
+    val argsStr = "--kafkaBootstrap 127.0.0.1:9092\n" +
+      "--kafkaTopic topic_test\n" +
+      "--kafkaGroupId topic_group\n" +
+      "--ckUrl jdbc:clickhouse://localhost:8123/default\n" +
+      "--ckUsername developer\n--ckPassword Mttproxy666\n" +
+      "--ckBatchSize 800\n" +
+      "--ckInsertSql 'insert into default.test values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'\n" +
+      "--checkpointType 'rocksdb'\n" +
+      "--checkpointInterval 5 \n" +
+      "--checkpointTimeOut 3"
+
+    //old
+    val oldProgramArgs = new ArrayBuffer[String]()
+    Try(argsStr.split("\\s+")).getOrElse(Array()).foreach(x => if (x.nonEmpty) oldProgramArgs += x)
+    println(oldProgramArgs)
+    //new
+    val newProgramArgs = new ArrayBuffer[String]()
+    val pattern = Pattern.compile("[^\\s\"']+|\"[^\"]*\"|'[^']*'")
+    val regexMatcher = pattern.matcher(argsStr)
+    while (regexMatcher.find()) {
+      newProgramArgs += regexMatcher.group().replaceAll("\"", "")
+    }
+    println(newProgramArgs)
```

Comment:
use Assertions.assertEquals determine whether it meets expectations,not println.

e.g:
```
Assertions.assertEquals("--ckUrl", newProgramArgs(7))
Assertions.assertEquals("jdbc:clickhouse://localhost:8123/default", newProgramArgs(8))
...
```